ci/jenkins/jobs/macros.yaml

```diff
@@ -302,3 +302,47 @@
            #!/bin/bash
            set -e
            ./ci/jenkins/stop-stale-jobs.sh --pull-request "${ghprbPullId}" --jenkins "${JENKINS_URL}"
+
+- builder:          
+    name: builder-kind-ipv6-ds-conformance
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          KIND_TIMEOUT=135
+          #  Delete all kind clusters created more than 135 mins ago. 135 minutes is the timeout 
+          #  we have configured for running conformance, NetworkPolicy and e2e tests on Kind, 
+          #  so clusters older than that can de deleted safely.
+          ./ci/kind/kind-setup.sh destroy --all --until "${{KIND_TIMEOUT}}"
+          ./ci/kind/kind-install.sh
+          ./ci/kind/kind-setup.sh --antrea-cni create "kind-${{BUILD_NUMBER}}" --ip-family '{ip_version}'
+          kind export kubeconfig -n "kind-${{BUILD_NUMBER}}" --kubeconfig ${{PWD}}/.kube/config
+          set +e
+          ./ci/jenkins/test.sh --testcase '{conformance_type}' --registry ${{DOCKER_REGISTRY}} --kubeconfig ${{PWD}}/.kube/config --testbed-type "kind" --enable-ipv6 '{ip_version}' --kind-cluster-name "kind-${{BUILD_NUMBER}}" --jenkins-location "{{jenkins_location}}"
```

Comment:
It's better to refactor existing builder-conformance-kind to avoid adding redundant jenkins builders. I am okay to add one more builder for e2e test, as currently there is no such builder.